src/PlaydateForm/PlaydateForm.js

```diff
@@ -11,7 +12,6 @@ class PlaydateForm extends Component {
   }
 
   componentDidMount = () => {
```

Comment:
Is a `componentDidMount` necessary to set the state here? Why not set the `parks` array in state to the `parksData` that you're importing? Maybe if you're going to fetch the data and the `componentDidMount` is there for future fetch calls.